### PR TITLE
Update bootstrap css to 5.2.3

### PR DESCRIPTION
**Pull request by ThallesVFS**
Update bootstrap css framework 4.4.1 to 5.2.3

### CODE REVIEW

**Comment by thalles** [comment]
image

**Comment by Xerkus** [comment]
Is there a reason for 5.1.3 when latest is 5.2.3?

**Review by Xerkus — CHANGES_REQUESTED** [review]
jquery was dropped, its reference also needs to be removed from layout.

**Comment by thalles** [comment]
Sorry! I will fix

File: module/Application/view/application/index/index.phtml [hunk 1]
```diff
@@ -3,27 +3,32 @@
...
-    <h1 class="display-4">Welcome to<br>Laminas MVC Skeleton Application</h1>
 
-    <p>
-        Congratulations! You have successfully installed the
-        <a href="https://github.com/laminas/laminas-mvc-skeleton" target="_blank">Laminas MVC Skeleton</a>.
-        This skeleton can serve as a simple starting point for you to begin
-        building your application on Laminas MVC.
-    </p>
+<div class="p-5 mb-4 bg-light rounded-3">
+    <div class="container-fluid py-5">
```
Comment (froschdesign) [inline module/Application/view/application/index/index.phtml]: Why is the container used here?

File: module/Application/view/application/index/index.phtml [hunk 2]
```diff
@@ -34,12 +39,14 @@
             </p>
 
             <a class="btn btn-primary" href="https://getlaminas.org/" target="_blank">Laminas Project Portal &raquo;</a>
```
Comment (froschdesign) [inline module/Application/view/application/index/index.phtml]: Please use a card footer for the buttons.

File: module/Application/view/application/index/index.phtml [hunk 3]
```diff
@@ -3,27 +3,32 @@
...
+    </div>
 </div>
 
-<div class="card-deck">
-    <div class="card">
-        <h2 class="card-header">Follow Development</h2>
-        <div class="card-body">
+
+<div class="row row-cols-1 row-cols-md-3 g-4">
+  <div class="col">
```
Comment (froschdesign) [inline module/Application/view/application/index/index.phtml]: Please use 4 spaces for indentation like above – everywhere.

**Review by froschdesign — CHANGES_REQUESTED** [review]

**Comment by thalles** [comment]
> Is there a reason for 5.1.3 when latest is 5.2.3?

Done!

**Review by Xerkus — APPROVED** [review]

**Comment by thalles** [comment]
Hi @all!
Please check again

File: module/Application/view/application/index/index.phtml (hunk 1 above)
Comment (froschdesign) [reply module/Application/view/application/index/index.phtml]: @thalles 
Can you please explain why the additional container is needed here?

File: module/Application/view/application/index/index.phtml (hunk 3 above)
Comment (froschdesign) [reply module/Application/view/application/index/index.phtml]: @thalles 
Please check again all indentations.

File: module/Application/view/application/index/index.phtml (hunk 1 above)
Comment (froschdesign) [reply module/Application/view/application/index/index.phtml]: So it seems there was no need.

**Comment by thalles** [comment]
Hey @everyone!
is anything still missing?

File: module/Application/view/application/index/index.phtml (hunk 3 above)
Comment (froschdesign) [reply module/Application/view/application/index/index.phtml]: @thalles 
Please check again all indentations like this.

File: module/Application/view/application/index/index.phtml [hunk 4]
```diff
@@ -32,26 +37,31 @@
...
+        </div>
+        <div class="card-footer">
             <a class="btn btn-primary" href="https://getlaminas.org/" target="_blank">Laminas Project Portal &raquo;</a>
         </div>
     </div>
+  </div>
 
-    <div class="card">
+  <div class="col">
+    <div class="card h-100">
```
Comment (froschdesign) [inline module/Application/view/application/index/index.phtml]: This one.

File: module/Application/view/application/index/index.phtml [hunk 5]
```diff
@@ -32,26 +37,31 @@
...
-                Explore Laminas Modules &raquo;
-            </a>
+                Explore Laminas Modules &raquo;</a>
         </div>
     </div>
+  </div>
 
-    <div class="card">
+  <div class="col">
+    <div class="card h-100">
```
Comment (froschdesign) [inline module/Application/view/application/index/index.phtml]: Also this.

File: module/Application/view/application/index/index.phtml [hunk 6]
```diff
@@ -32,26 +37,31 @@
                 (link requires a GitHub account). This is a great resource
                 for staying up to date with the latest developments!
             </p>
-
+        </div>
+        <div class="card-footer">
             <a class="btn btn-primary" href="https://getlaminas.org/" target="_blank">Laminas Project Portal &raquo;</a>
         </div>
     </div>
+  </div>
```
Comment (froschdesign) [inline module/Application/view/application/index/index.phtml]: This.

File: module/Application/view/application/index/index.phtml [hunk 7]
```diff
@@ -61,8 +71,10 @@
                 We'd love to hear any questions or feedback you may have
                 regarding any release.
             </p>
-
+        </div>
+        <div class="card-footer">
             <a class="btn btn-primary" href="https://laminas.dev/chat" target="_blank">Ping us on chat &raquo;</a>
         </div>
     </div>
+  </div>
```
Comment (froschdesign) [inline module/Application/view/application/index/index.phtml]: And here.

**Review by froschdesign — CHANGES_REQUESTED** [review]

**Review by froschdesign — CHANGES_REQUESTED** [review]
The `data` attributes have changed with version 5 of Bootstrap and need to be updated. Otherwise the [toogler for the navbar](https://getbootstrap.com/docs/5.2/components/navbar/#toggler) will not work.

https://getbootstrap.com/docs/5.2/migration/#javascript

**Comment by ThallesVFS** [comment]
> The `data` attributes have changed with version 5 of Bootstrap and need to be updated. Otherwise the [toogler for the navbar](https://getbootstrap.com/docs/5.2/components/navbar/#toggler) will not work.
> 
> https://getbootstrap.com/docs/5.2/migration/#javascript

Done!

**Comment by ThallesVFS** [comment]
Hi @froschdesign, let me know if missing something.
Thanks!

**Comment by froschdesign** [comment]
@ThallesVFS 
When you are finished, mark this pull request as "Ready for review". In the meantime, I will test the changes locally.

**Comment by ThallesVFS** [comment]
Sorry, it is my first time contributing for a open source on github.
Thanks for your help!

**Review by froschdesign — APPROVED** [review]

**Comment by Xerkus** [comment]
Thank you.

**Pull request merged**